ui/app/components/app/connected-accounts-list/index.scss

```diff
@@ -115,6 +115,18 @@
     i {
       padding-right: 8px;
       font-size: 18px;
+      color: #C4C4C4;
```

Comment:
Is there an existing variable that we could use instead? Or can we add this to `ui/app/css/itcss/settings/variables.scss`? (cc @rachelcope)

Reply:
We've got _a lot_ of greys:

```scss
$Grey-000: #f2f3f4;
$Grey-100: #D6D9DC;
$Grey-800: #c4c4c4;
$Grey-200: #bbc0c5;
$Grey-300: #9fa6ae;
$Grey-400: #848c96;
$Grey-200: #bbc0c5;
$Grey-500: #6A737D;
$Grey-600: #535a61;
$Grey-800: #24272a;
```

Reply:
somehow missed Grey-800 :+1: